tests/Feature/Community/ShortcodeTest.php

```diff
@@ -334,4 +341,485 @@ public static function youtubeUrlProvider(): array
             ['http://www.youtube.com/watch?v=1tNKq6KWPhY', '//www.youtube-nocookie.com/embed/1tNKq6KWPhY'],
         ];
     }
+
+    public function testConvertToMarkdownBold(): void
+    {
+        $this->assertSame(
+            '**Hello**',
+            Shortcode::convertToMarkdown('[b]Hello[/b]')
+        );
+    }
+
+    public function testConvertToMarkdownItalic(): void
+    {
+        $this->assertSame(
+            '*Hello*',
+            Shortcode::convertToMarkdown('[i]Hello[/i]')
+        );
+    }
+
+    public function testConvertToMarkdownStrikethrough(): void
+    {
+        $this->assertSame(
+            '~~Hello~~',
+            Shortcode::convertToMarkdown('[s]Hello[/s]')
+        );
+    }
+
+    public function testConvertToMarkdownUnderline(): void
+    {
+        $this->assertSame(
+            '__Hello__',
+            Shortcode::convertToMarkdown('[u]Hello[/u]')
+        );
+    }
+
+    public function testConvertToMarkdownCode(): void
+    {
+        $this->assertSame(
+            '```print("Hello")```',
+            Shortcode::convertToMarkdown('[code]print("Hello")[/code]')
+        );
+    }
+
+    public function testConvertToMarkdownQuote(): void
+    {
+        $this->assertSame(
+            '> This is a quote',
+            Shortcode::convertToMarkdown('[quote]This is a quote[/quote]')
+        );
+    }
+
+    public function testConvertToMarkdownMultilineQuote(): void
+    {
+        $this->assertSame(
+            "> Line 1\n> Line 2\n> Line 3",
+            Shortcode::convertToMarkdown("[quote]Line 1\nLine 2\nLine 3[/quote]", 10000, preserveWhitespace: true)
+        );
+    }
+
+    public function testConvertToMarkdownNestedQuotes(): void
+    {
+        $this->assertSame(
+            '> > Inner quote Outer quote', // nested quotes get prefixed together
```

Comment:
This seems wrong. I would expect:
```
> > Inner quote
> Outer quote
```

But that might be because there isn't a newline between the inner and outer quote.

Additionally, Discord doesn't seem to do nested quotes, so it's probably good enough.